model/user_task_annotations/db.go

```diff
@@ -0,0 +1,85 @@
+package user_task_annotations
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/anser/bsonutil"
+	adb "github.com/mongodb/anser/db"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+var (
+	// bson fields for the UserTaskAnnotation struct
+	IdKey              = bsonutil.MustHaveTag(UserTaskAnnotation{}, "Id")
+	TaskIdKey          = bsonutil.MustHaveTag(UserTaskAnnotation{}, "TaskId")
+	TaskExecutionKey   = bsonutil.MustHaveTag(UserTaskAnnotation{}, "TaskExecution")
+	NoteKey            = bsonutil.MustHaveTag(UserTaskAnnotation{}, "Note")
+	IssuesKey          = bsonutil.MustHaveTag(UserTaskAnnotation{}, "Issues")
+	SuspectedIssuesKey = bsonutil.MustHaveTag(UserTaskAnnotation{}, "SuspectedIssues")
+	SourceKey          = bsonutil.MustHaveTag(UserTaskAnnotation{}, "Source")
+)
+
+const Collection = "user_task_annotation"
+
+// FindOne gets one UserTaskAnnotation for the given query.
+func FindOne(query db.Q) (UserTaskAnnotation, error) {
```

Comment:
Conventionally FindOne and Find check adb.ResultsNotFound, rather than each individual function using it